default/src/components/signer.js

```diff
@@ -120,6 +128,139 @@ const GetTxParams = () => {
   );
 };
 
+const SendASA = () => {
+  let asaData = YAMLData.default.default.asa;
+  let deployedASAName;
+  let deployedASAId;
+  let asaId;
+  for (const [key, value] of Object.entries(asaData)) {
+    deployedASAName = key;
+    deployedASAId = value.assetIndex;
```

Comment:
Aren't we overwriting asa name, id each time in loop?

Reply:
Yes. But as this is just an example, so we have only 1 `asaDef` in `asaData`.

Reply:
My question is why do we use a loop here, if we overwrite variable each time

Reply:
I'll update the variables to a list.